sos/policies/distros/uniontech.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) Red Hat, Inc. 2019
```

Comment:
This copyright claim seems incorrect - please update it with a correct copyright holder for this new file.

Reply:
@bmr-cymru 
I have revised this question. Thank you very much.